war/src/main/webapp/help/system-config/globalEnvironmentVariables.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    These key-value pairs apply for every build on every node. They can be used in Jenkins'
-    configuration (as $key or ${key}) and will be added to the environment for processes launched from the build.
+    These key-value pairs apply for every build on every node. They can be used in Jenkins&rsquo;s
```

Comment:
As the main rule is keep consensus and we already have a lot of documentation using Jenkins', this would be the right form in this case.

```suggestion
    These key-value pairs apply for every build on every node. They can be used in Jenkins&rsquo;
```